language-design.md

```diff
@@ -62,6 +102,12 @@ the commenters?
 
 **Response**
 
+The essence of the conflict is that tools which are easy-to-use for novices might not be the best tools for experts. Many of the commenters would trade program readability for efficiency (usually programming speed or runtime speed). [[Unclemilford, 2012](https://www.fastcodesign.com/1665735/why-arent-computer-programming-languages-designed-better#comment-f7558800-41fb-11e1-921b-9794fc737ae8); [STRENGTHINESS, 2015](https://www.fastcodesign.com/1665735/why-arent-computer-programming-languages-designed-better#comment-93e75ca0-9b5b-11e4-a3e4-4930d33952b2)].
+
+We both think that both the article and the comments raise some good points. The article asks "Why shouldn’t [programming languages] be humanely designed?" [Pavlus, 2012], and strongly implies that the answer is "they should".
+However, it makes the implicit assumption that it should be humanely designed for "novice programmers". The comments point out that the needs of novice programmers aren't the needs of professional programmers, but also downplay the importance of usability to those professional programmers (for example, see [STRENGTHINESS, 2015](https://www.fastcodesign.com/1665735/why-arent-computer-programming-languages-designed-better#comment-93e75ca0-9b5b-11e4-a3e4-4930d33952b2)]). We feel like usability is still important to professional programmers, but that the methods for making programming languages usable for this specific group are different from the ones for the general public.
```

Comment:
What types of features/decisions make a language more appropriate for novices? What makes a language more appropriate for experienced users?

Are these parties always at odds?

I definitely agree that it is  a frequent conflict :smile: